### ISSUE TITLE
Browser compatibility

### ISSUE THREAD

**Issue by iteles** [research-resource/research_resource#23]
What are the client requirements for browser/device compatibility?

**Comment by sarahwelton** [comment]
We need to find out from them what browsers and devices they have access to at work. As it's NHS it might be limited.

**Comment by sarahwelton** [comment]
From the team at RR: "Most of us are on PCs and use Chrome - we aren't limited to Internet Explorer. We do need IT admin to install new programmes/browsers on our PCs. I guess we update as and when."

**Comment by iteles** [comment]
@sarahwelton Good call, thanks for that. Does that mean the latest version of Chrome and IE on desktop are all they're after?

Are all of their users also going to be using NHS computers?

**Comment by ellemindwave** [comment]
@iteles @Cleop Yes, latest version of Chrome and IE for now. 
Their users won't just be using NHS computers, they will be using their home computers/mobile devices etc too

**Comment by iteles** [comment]
@ellemindwave Thanks. So _for now_ the request from the client is to accommodate _only_ the latest versions of Chrome and IE and then update things at a later date for end users as a result of user testing?
